experiment/coverage_utils.py

```diff
@@ -292,3 +406,170 @@ def extract_covered_regions_from_summary_json(summary_json_file):
     except Exception:  # pylint: disable=broad-except
         logger.error('Coverage summary json file defective or missing.')
     return covered_regions
+
+
+def prepare_name_dataframes(df_container):
+    """Populating DataFrames with experiment specific
```

Comment:
Tense for this docstring seems off, maybe "populates"?
Also, the linebreak here seems arbitrary, typically you should try to fit as many words a possible on a line in a comment.
Many editors can do this for you (in emacs I think you highlight the comment and do M-;).